spec.go

```diff
@@ -37,13 +37,21 @@ func (c *SpecController) Show(ctx *app.ShowSpecContext) error {
 	if len(elems) < 3 {
 		return fmt.Errorf("invalid package path %s", packagePath)
 	}
+	var branch string
+	last := elems[len(elems)-1]
+	if strings.Contains(last, ":") {
+		branch = strings.Split(last, ":")[1]
+		design := strings.Split(last, ":")[0]
```

Comment:
Not a huge deal but this code could be written more efficiently by avoiding to call `Contains` then `Split` twice. Something like:
```go
var branch string
parts := strings.Split(elems[len(elems)-1], "@")
design := parts[0]
if len(parts) > 1 {
        branch = parts[1]
}
```